README.md

```diff
@@ -24,12 +24,34 @@ ChatGPT uses your private conversations history to train its models. [OpenAI dis
 
 # Table of Contents
 
+- [Configuration](#configuration)
 - [Deploy the app on Render](#deploy-the-app-on-render)
 - [Deploy the app on Fly.io](#deploy-the-app-on-fly)
 - [Contribute as a developer](#contribute-as-a-developer)
 - [Understanding the Docker configuration](#understanding-the-docker-configuration)
 - [Changelog](#changelog)
 
+# Configuration
+
+Following features can be configured with environment variables:
+
+- `REGISTRATON_FEATURE` - Set to `false` to disable registration.
+  Default is `true`.
+- `AUTHENTICATION_APPROACH` - Set to `google` to use Google OAuth for authentication, `http_header` to use HTTP header authentication, or `password` to use password authentication.
+  Default is `password`.
+- `AUTHENTICATION_HTTP_HEADER_EMAIL` - Name of the HTTP request header, which can be used to derive an email address of a new user.
+  Required if `AUTHENTICATION_APPROACH` is set to `http_header`.
+  Default is `Tailscale-User-Login`.
+- `AUTHENTICATION_HTTP_HEADER_NAME` - Name of the HTTP request header, which can be used to derive the name of a new user.
+  Required if `AUTHENTICATION_APPROACH` is set to `http_header`.
+  Default is `Tailscale-User-Name`.
```

Comment:
This is what I would like to remove. This would mean email and name will not be set.